nifi-nar-bundles/nifi-extension-utils/nifi-hadoop-utils/src/main/java/org/apache/nifi/processors/hadoop/AbstractHadoopProcessor.java

```diff
@@ -257,9 +260,63 @@ public final void abstractOnScheduled(ProcessContext context) throws IOException
 
     @OnStopped
     public final void abstractOnStopped() {
```

Comment:
Is there anything common to other processors here?  Are we going to end up pointing other processors with this problem, one after another to this code?